File: JustLanguage/Entities/ArticleInfo.cs
namespace JustLanguage.Entities;

public class ArticleInfo
{
    /// <summary>
    /// Id
    /// </summary>
    public int Id { get; set; }

    /// <summary>
    /// article title
    /// </summary>
    public string Title { get; set; } = string.Empty;

    /// <summary>
    /// article content
    /// </summary>
    public string Content { get; set; } = string.Empty;

    /// <summary>
    /// article cover image base64 string
    /// </summary>
    public string CoverImageBase64 { get; set; } = string.Empty;

    /// <summary>
    /// article author
    /// </summary>
    public string Author { get; set; } = string.Empty;

    /// <summary>
    /// source url
    /// </summary>
    public string SrcUrl { get; set; } = string.Empty;

    /// <summary>
    /// date time
    /// </summary>
    public DateTime UploadTime { get; set; }
}

File: Web/src/Models/ArticleInfo.ts
class ArticleInfo {
  id: number
  title: string
  content: string
  coverImageBase64: string
  author: string
  srcUrl: string
  constructor() {
    this.id = 0
    this.title = ''
    this.content = ''
    this.coverImageBase64 = ''
    this.author = ''
    this.srcUrl = ''
  }
}
export { ArticleInfo }
